frontend/src/components/userBanner/index.js

```diff
@@ -17,11 +17,21 @@ class UserBanner extends Component {
       upBanner: null,
       draggingOverAvatar: false,
       draggingOverBanner: false,
-      following: this.props.user.following || false,
+      following: this.props.user.following,
+      followsYou: this.props.user.followsYou,
       banUser: false
     }
   }
 
+  componentDidUpdate(prevProps) {
+    if (this.props.user.username !== prevProps.user.username) {
+      this.setState({
+        following: this.props.user.following,
+        followsYou: this.props.user.followsYou
+      })
+    }
```

Comment:
```suggestion
    const { username, following, followsYou } = this.props
    
    if (username !== username) {
      this.setState({
        following,
        followsYou
      })
    }
```

Deconstruct and if I'm correct, state doesn't have to specify var name if variable is same as state property name